source/includes/administration/_invoices.md

```diff
@@ -31,7 +31,7 @@ curl "https://cloudmc_endpoint/rest/invoices/find/efd32752-c6f2-45cf-b494-cc6be8
 			"discounted": true,
 			"discounts": [],
 			"startDate": "2021-07-01T00:00:00Z",
-			"endDate": "2021-08-01T00:00:00Z",
+			"endDate": "2021-07-31T00:00:00Z",
```

Comment:
The API should return the end of the day right?

Reply:
That's the thing as we are only subtracting a day so the time part won't be `23:59:59Z` of 31st July. That is why I added the description part as well saying that the endDate is inclusive.

Reply:
but im saying it should be inclusive no? The data included in an invoice doesnt stop at 07-31 at midnight does it?. It stops at `07-31:23:59:59`